### PR TITLE
Build libayatana-common with ENABLE_LOMIRI_FEATURES

### PR DESCRIPTION
**Pull request by tari01**

### CODE REVIEW

File: .build.yml [hunk 1]
```diff
@@ -264,7 +268,7 @@ before_scripts:
 build_scripts:
   - if [ ${DISTRO_NAME} == "debian" ];then
   -     export CFLAGS+=" -Wsign-compare -Wunused-parameter"
-  -     cppcheck --enable=warning,style,performance,portability,information,missingInclude .
+  -     cppcheck --enable=warning,performance,portability,information --suppress=missingInclude .
```
Comment (sunweaver) [inline .build.yml:271]: Why do you drop the style check? This change sneakily does two things at once without explaining.

**Review by sunweaver — CHANGES_REQUESTED** [review]
@tari01: Please revisit the cppcheck adjustment. Thanks.

File: .build.yml (hunk 1 above)
Comment (tari01) [reply .build.yml]: Fixed in additional commits.

**Review by sunweaver — APPROVED** [review]
Very good. Cool that you fixed the cppcheck style part...

**Pull request merged**